projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml

```diff
@@ -59,6 +59,20 @@ frontend-tests-vdk-jupyterlab-extension:
     reports:
       junit: "projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/frontend_tests_results.xml"
 
+integration-tests-vdk-jupyterlab-extension:
+  extends: .common-setup-vdk-jupyterlab-extension
+  stage: pre_release_test
```

Comment:
You know that you can change the image wiht `image:` tag, right?

  The default image use is `python:3.8-alpine` I can see from teh logs. 

But the warning in the logs says that it isn't supported. 
So you can change it to normal Ubuntu image or something else that is supported by playwright

Reply:
apparently the 'not supported' warning isn't a massive issue according to what I read